src/oncall/ui/static/js/oncall.js

```diff
@@ -865,8 +873,13 @@ var oncall = {
       renderTeamSummary: function(data){
         var template = Handlebars.compile(this.data.cardOncallTemplate),
             $container = this.data.$page.find('#oncall-now-container');
-
+        data.showEscalate = oncall.data.user && this.data.teamData.iris_plan
+          && window.appData.iris_plan_settings.activated;
```

Comment:
we shouldn't be relying on appData here. that was a hacky thing we did for iris because we had a few weeks to build it and it wasn't an SPA. we wanna get away from this in oncall, i think this feature would be best in its own function under oncall.team.calendar.escalate where you make a call to the api to get the data async then renders the button/sets up modal.